### PR TITLE
ADR-80: CEE diagnostic gathering mechanism

### PR DESCRIPTION
**Pull request by k-wall**
Draft and placeholder for ADR 78

Signed-off-by: kwall <kwall@apache.org>

### CODE REVIEW

File: _adr/80/index.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+num: 80
+title: "CEE diagnostic gathering mechanism"
+status: "Draft"
+authors:
+  - "Keith Wall""
+---
+
+## Context and Problem Statement
+
+In order to proviide support to customers of a Managed Service, CEE require a mechansim that allows them to capture the
```
Comment (tombentley) [inline _adr/80/index.adoc:11]:
```suggestion
In order to provide support to customers of a Managed Service, CEE require a mechanism that allows them to capture the
```

File: _adr/80/index.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+state of the instance of the service for diagnostic purposes.  CEE will use this information in conjunction with the
+problem statement provided by the customer in order to efficiently investigate the problem and determine a solution.
+
+The data gathering mechanism must consider the issue of Personal Information (PI) or other sensitive information and ensure
+that is its handled appropiately.
```
Comment (tombentley) [inline _adr/80/index.adoc:16]:
```suggestion
that it is handled appropriately.
```

**Review by tombentley — APPROVED** [review]
A few typos, but LGTM.

**Comment by k-wall** [comment]
@emmanuelbernard would you mind merging?

**Comment by emmanuelbernard** [comment]
I will merge it but I think it needs a different title.

Since these are open source, any obscure function name is not helping.
I propose to rename `CEE` with `a support organization`.

The one other question I had was whether it was java diagnosing or generic diagnosing mechanism regardless of technology? That might also influence the title.

CC @k-wall 

@tombentley where can we capture these generic writing guidelines?
Anther one is to one line per sentence model that we can discuss but that I've found useful in markdown and asciidoc documents https://emmanuelbernard.com/blog/2014/10/06/one-line-per-ide-feedback/

Finally where should I bring these feedback on a proto ADR? An issue created already? Here in the PR about to be merged?

**Comment by tombentley** [comment]
@emmanuelbernard

> @tombentley where can we capture these generic writing guidelines?

 [pages/about/adr-process.adoc](https://github.com/bf2fc6cc711aee1a0c2a/architecture/blob/main/pages/about/adr-process.adoc) is probably the best place for now, but we might want to have separate pages for the ADR process vs the ADR content. I've opened https://github.com/bf2fc6cc711aee1a0c2a/architecture/issues/36 to track the need for this doc.

> Finally where should I bring these feedback on a proto ADR? An issue created already? Here in the PR about to be merged?

I've opened https://github.com/bf2fc6cc711aee1a0c2a/architecture/issues/35 if that's what you mean by proto ADR? If so, feel free to comment there.

**Comment by emmanuelbernard** [comment]
> I've opened #35 if that's what you mean by proto ADR? If so, feel free to comment there.


No I meant where / how to discuss the content of ADR-80. Sorry for my "creative" namings.

**Comment by tombentley** [comment]
Comments on the PR then. I think issues in this repo should be limited to: 

* Proposed ADRs
* Issues for the website itself (tracking tasks we want to get to eventually)

**Pull request merged**